From 1a7435d2cc95ea67e9361582207e6e82cf5ba207 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 11 Dec 2018 08:35:26 +0100
Subject: [PATCH] Fix include path for qSlicerBaseQTGUI

---
 CMake/SlicerInstallConfig.cmake.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMake/SlicerInstallConfig.cmake.in b/CMake/SlicerInstallConfig.cmake.in
index 41e3a2caf..f78c34220 100644
--- a/CMake/SlicerInstallConfig.cmake.in
+++ b/CMake/SlicerInstallConfig.cmake.in
@@ -94,7 +94,7 @@ set(Slicer_Base_LIBRARY_DIRS
 set(Slicer_Base_INCLUDE_DIRS
   "${Slicer_INSTALL_PREFIX}/@Slicer_INSTALL_INCLUDE_DIR@"
   "${Slicer_INSTALL_PREFIX}/@Slicer_INSTALL_INCLUDE_DIR@/SlicerBaseCLI"
-  "${Slicer_INSTALL_PREFIX}/@Slicer_INSTALL_INCLUDE_DIR@/SlicerBaseGUI"
+  "${Slicer_INSTALL_PREFIX}/@Slicer_INSTALL_INCLUDE_DIR@/qSlicerBaseQTGUI"
   "${Slicer_INSTALL_PREFIX}/@Slicer_INSTALL_INCLUDE_DIR@/SlicerBaseLogic"
   )
 
-- 
2.19.2

