From 38a5eb5826fa40f81fab6c613dec4e18f8628791 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 11 Dec 2018 13:09:34 +0100
Subject: [PATCH] Include Qt Xml in Slicer build logic macro

---
 CMake/SlicerMacroBuildModuleLogic.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMake/SlicerMacroBuildModuleLogic.cmake b/CMake/SlicerMacroBuildModuleLogic.cmake
index 2114d4fa1..7160c8e27 100644
--- a/CMake/SlicerMacroBuildModuleLogic.cmake
+++ b/CMake/SlicerMacroBuildModuleLogic.cmake
@@ -49,6 +49,7 @@ macro(SlicerMacroBuildModuleLogic)
   endif()
 
   list(APPEND MODULELOGIC_INCLUDE_DIRECTORIES
+    ${Qt5Xml_INCLUDE_DIRS}
     ${Slicer_Libs_INCLUDE_DIRS}
     ${Slicer_Base_INCLUDE_DIRS}
     ${Slicer_ModuleLogic_INCLUDE_DIRS}
-- 
2.19.2

