From d40914fdf0ae60bdf07f490cdb6f00657c5ae1fb Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Mon, 10 Dec 2018 12:24:34 +0100
Subject: [PATCH] Including VTK and ITK in SlicerUse

---
 CMake/UseSlicer.cmake.in | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/CMake/UseSlicer.cmake.in b/CMake/UseSlicer.cmake.in
index 936932ee5..1b837a3a4 100644
--- a/CMake/UseSlicer.cmake.in
+++ b/CMake/UseSlicer.cmake.in
@@ -145,6 +145,18 @@ endif()
 # --------------------------------------------------------------------------
 find_package(Qt5 COMPONENTS Core Widgets REQUIRED)
 
+# --------------------------------------------------------------------------
+# VTK
+# --------------------------------------------------------------------------
+find_package(VTK REQUIRED)
+include(${VTK_USE_FILE})
+
+# --------------------------------------------------------------------------
+# ITK
+# --------------------------------------------------------------------------
+find_package(ITK REQUIRED)
+include(${ITK_USE_FILE})
+
 # Set '<PROJECTNAME>_DIR' variable(s) so that find_package(<PROJECTNAME>) works as expected.
 # NA
 
-- 
2.19.2

