From 2cce31b80ac22b9913178e0f5b70a356d6c0585e Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 11 Dec 2018 14:06:39 +0100
Subject: [PATCH] Include VTK in slicer build logic macro

---
 CMake/SlicerMacroBuildModuleLogic.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMake/SlicerMacroBuildModuleLogic.cmake b/CMake/SlicerMacroBuildModuleLogic.cmake
index 7160c8e27..7b8c85f2d 100644
--- a/CMake/SlicerMacroBuildModuleLogic.cmake
+++ b/CMake/SlicerMacroBuildModuleLogic.cmake
@@ -49,6 +49,7 @@ macro(SlicerMacroBuildModuleLogic)
   endif()
 
   list(APPEND MODULELOGIC_INCLUDE_DIRECTORIES
+    ${VTK_INCLUDE_DIRS}
     ${Qt5Xml_INCLUDE_DIRS}
     ${Slicer_Libs_INCLUDE_DIRS}
     ${Slicer_Base_INCLUDE_DIRS}
-- 
2.19.2

