From ce1c2975a5586180057d73fba310b019619cf219 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 11 Dec 2018 10:42:46 +0100
Subject: [PATCH] Include CTK libraries for linking Loadable Modules

---
 CMake/SlicerMacroBuildLoadableModule.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMake/SlicerMacroBuildLoadableModule.cmake b/CMake/SlicerMacroBuildLoadableModule.cmake
index 9a709bd9c..c7674ccc9 100644
--- a/CMake/SlicerMacroBuildLoadableModule.cmake
+++ b/CMake/SlicerMacroBuildLoadableModule.cmake
@@ -237,6 +237,7 @@ macro(slicerMacroBuildLoadableModule)
     ${Slicer_GUI_LIBRARY}
     ${Slicer_CORE_LIBRARY}
     ${LOADABLEMODULE_TARGET_LIBRARIES}
+    ${CTK_LIBRARIES}
     )
 
   # Apply user-defined properties to the library target.
-- 
2.19.2

