From 1e661142c70bf510027d58d3f36365ae415a6858 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 11 Dec 2018 11:53:43 +0100
Subject: [PATCH] Include Xml in Qt5 components

---
 CMake/UseSlicer.cmake.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMake/UseSlicer.cmake.in b/CMake/UseSlicer.cmake.in
index 1f00d6ffe..d17434e42 100644
--- a/CMake/UseSlicer.cmake.in
+++ b/CMake/UseSlicer.cmake.in
@@ -143,7 +143,7 @@ endif()
 # --------------------------------------------------------------------------
 # Qt
 # --------------------------------------------------------------------------
-find_package(Qt5 COMPONENTS Core Widgets REQUIRED)
+find_package(Qt5 COMPONENTS Core Widgets Xml REQUIRED)
 
 # --------------------------------------------------------------------------
 # VTK
-- 
2.19.2

