From 7ba51a345f91e347fa33974bdb3dbdc4dee38386 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Wed, 12 Dec 2018 11:54:30 +0100
Subject: [PATCH] Add instal qtloadablemodules include dire to build logic
 macro

---
 CMake/SlicerMacroBuildModuleLogic.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMake/SlicerMacroBuildModuleLogic.cmake b/CMake/SlicerMacroBuildModuleLogic.cmake
index 7b8c85f2d..69ae08471 100644
--- a/CMake/SlicerMacroBuildModuleLogic.cmake
+++ b/CMake/SlicerMacroBuildModuleLogic.cmake
@@ -55,6 +55,7 @@ macro(SlicerMacroBuildModuleLogic)
     ${Slicer_Base_INCLUDE_DIRS}
     ${Slicer_ModuleLogic_INCLUDE_DIRS}
     ${Slicer_ModuleMRML_INCLUDE_DIRS}
+    ${Slicer_INSTALL_QTLOADABLEMODULES_INCLUDE_DIR}
     )
 
   if(Slicer_BUILD_CLI_SUPPORT)
-- 
2.19.2

