From a9283bbaa85e173c96a82977430f44c01be03a7d Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Mon, 10 Dec 2018 15:44:35 +0100
Subject: [PATCH] Adding MRML libraries to SlicerConfig template

---
 CMake/SlicerInstallConfig.cmake.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMake/SlicerInstallConfig.cmake.in b/CMake/SlicerInstallConfig.cmake.in
index 096388087..41e3a2caf 100644
--- a/CMake/SlicerInstallConfig.cmake.in
+++ b/CMake/SlicerInstallConfig.cmake.in
@@ -150,3 +150,6 @@ set(CMAKE_MODULE_PATH
 
 
 set(CTK_QT_VERSION "@CTK_QT_VERSION@")
+
+# MRML libraries - This variable regroup all related MRML libraries
+set(MRML_LIBRARIES "@MRML_LIBRARIES@")
-- 
2.19.2

