From bfa4740eeb27ade541db5bb2245c359510772712 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Tue, 11 Dec 2018 07:45:28 +0100
Subject: [PATCH] Fix setting install development flag in Logic

---
 Base/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Base/CMakeLists.txt b/Base/CMakeLists.txt
index 87b56a167..579c2a165 100644
--- a/Base/CMakeLists.txt
+++ b/Base/CMakeLists.txt
@@ -7,6 +7,8 @@ include_directories(
   ${CMAKE_CURRENT_BINARY_DIR}/QTCore
   )
 
+set(SlicerBaseLogic_INSTALL_NO_DEVELOPMENT ${Slicer_INSTALL_NO_DEVELOPMENT})
+
 add_subdirectory(Logic)
 add_subdirectory(QTCore)
 add_subdirectory(QTGUI)
-- 
2.19.2

